docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -56,10 +57,10 @@ export default function WhatToExpect() {
               </Typography>
             </Box>
             <Typography variant="body2" color="text.secondary">
-              The licenses are sold on a per-developer basis. The Pro plan includes a cap at 10
-              developers, extra developers do not need to be licensed. You can contact{' '}
-              <Link href="mailto:sales@mui.com">sales</Link> for a volume discount when licensing
-              over 50 developers under the Premium plan.
+              The licenses are sold on a per-developer basis, using the software directly or
+              indirectly. The Pro plan includes a cap at 10 developers, extra developers do not need
+              to be licensed. You can contact <Link href="mailto:sales@mui.com">sales</Link> for a
```

Comment:
```suggestion
              You can contact <Link href="mailto:sales@mui.com">sales</Link> for a
```

Reply:
To fix the confusion

```suggestion
              to pay for their seats. You can contact <Link href="mailto:sales@mui.com">sales</Link> for a
```